### ISSUE TITLE
Focus of Textfield, Dropdown, Combobox, Split Button and Spin Button are not accessible

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#9348]
Current focus uses only a color change to denote focus. This makes it difficult or impossible for many people to track using keyboard. Design pending.

**Comment by micahgodbolt** [comment]
@betrue-final-final do you know if we have designs for this yet? Shouldn't be difficult work to do once we have those new colors.

**Comment by dzearing** [comment]
@betrue-final-final any update?

**Comment by micahgodbolt** [comment]
yup...still an issue

seems stardust does this:

![image](https://user-images.githubusercontent.com/1434956/65109195-f3c60300-d98c-11e9-8dbb-eea58b60df1d.png)

and fast DNA increases border width:

![image](https://user-images.githubusercontent.com/1434956/65109310-04767900-d98d-11e9-87e3-271cf872d8d2.png)

**Comment by micahgodbolt** [comment]
fixed by #10885
